azurelinuxagent/common/utils/processutil.py

```diff
@@ -16,16 +16,13 @@
 #
 # Requires Python 2.6+ and Openssl 1.0+
 #
-import multiprocessing
+import select
```

Comment:
We might need to confirm if this is out of box. (I am certain `select` should be)

Reply:
DCR would have caught that (if it required installing an extra dependency), I think.